terracumber_config/tf_files/SUSEManager-Test-Ion-NUE.tf

```
@@ -130,90 +130,77 @@ module "cucumber_testsuite" {
     server_containerized = {
       provider_settings = {
         mac = "aa:b2:93:01:00:41"
-        memory = 12288
+        vcpu = 8
+        memory = 32768
       }
+      main_disk_size = 500
+      login_timeout = 28800
+      runtime = "podman"
+      container_repository = "registry.suse.de/devel/galaxy/manager/head/containerfile"
```

This should be pointing to our `TEST:Ion` repository:

```suggestion
      container_repository = "registry.suse.de/devel/galaxy/manager/test/ion/containerfile"
```